### PR TITLE
Kotlin Migration TimePreferenceTest 

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin Migration Of TimePreference Test

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.kt [hunk 1]
```diff
@@ -1,56 +1,43 @@
 /**
  * Copyright (c) 2021 Diego Rodriguez <diego.vincent.rodriguez@gmail.com>
- * 
- * 
+ *
```
Comment (lukstbit) [inline AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.kt:3]: When doing a migration the copyright header shouldn't be changed. Please revert all changes done to this copyright block.

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Review by david-allison — COMMENTED** [review]
Happy to merge once copyright is fixed

**Comment by Prince-kushwaha** [comment]
@david-allison copyright is fixed

File: AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.kt (hunk 1 above)
Comment (Prince-kushwaha) [reply AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.kt:3]: it is fixed

**Comment by david-allison** [comment]
> @david-allison copyright is fixed

There should be no diff on the copyright, it's still showing: https://github.com/ankidroid/Anki-Android/pull/11666/commits/8cd00a482e9c69ffe4ad1906e5b4f9e8a277eebf

**Comment by Prince-kushwaha** [comment]
how i revert all changes done to this copyright block?

**Comment by lukstbit** [comment]
@Prince-kushwaha 
- copy the copyright header from before(here's the [current file](https://github.com/ankidroid/Anki-Android/blob/7082f6ee93ed165180b935dbca44598a6566b3cc/AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.java) that you can use) and paste it instead of the copyright that you changed from your PR
- use `git commit --amend` to change the last commit in this PR 
- then force push this PR

**Comment by Prince-kushwaha** [comment]
@lukstbit [](https://github.com/ankidroid/Anki-Android/blob/7082f6ee93ed165180b935dbca44598a6566b3cc/AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.java)[current file](https://github.com/ankidroid/Anki-Android/blob/7082f6ee93ed165180b935dbca44598a6566b3cc/AnkiDroid/src/test/java/com/ichi2/preferences/TimePreferenceTest.java) copyright header  and my  kotlin file copyright header is same

**Comment by Prince-kushwaha** [comment]
@lukstbit i apply [this method](https://github.com/ankidroid/Anki-Android/pull/11666#issuecomment-1179674242) to revert copyright header but it not work because  java file copyright header and my kotlin file copyright header is same

**Comment by Prince-kushwaha** [comment]
I not understand why git show difference

**Review by lukstbit — COMMENTED** [review]
The copyright still shows a diff but I'm approving, there's no need to get hung up on this with such a small class.

**Review by lukstbit — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
yeah, no reason to hang up on spacing for the copyright on a small test file, since the copyright *is* still correct, let's get this in + done. Thanks for the help @Prince-kushwaha

**Pull request merged**